src/smartpeak/source/core/SampleGroupProcessor.cpp

```diff
@@ -32,7 +32,55 @@ namespace SmartPeak
 {
   ParameterSet MergeInjections::getParameterSchema() const
   {
-    return ParameterSet();
+    std::map<std::string, std::vector<std::map<std::string, std::string>>> param_struct({
+    {"MergeInjections", {
+      {
+        {"name", "scan_polarity_merge_rule"},
+        {"type", "string"},
+        {"value", "WeightedMean"},
+        {"description", ""},
```

Comment:
"The method used to merge the different feature values.  Options include `Sum` (the sum of all feature values), `Min` (the smallest feature value), `Max` (the largest feature value), `Mean` (the mean of all feature values), and `WeightedMean` (the weight average of all feature values)."